lib/models/canvas_object.rb

```diff
@@ -0,0 +1,25 @@
+class CanvasObject < Forgery
+
+  def self.push_csv_to_canvas(opts={})
+    if(opts.nil? || opts[:host].nil? || opts[:token].nil?)
+      raise 'Please provide options hash with both :host and :token'
+    end
+    if(!File.file?(opts[:file]))
+      raise 'Please gen_file before trying to push to canvas'
+    end
+    uri = URI.parse("https://#{opts[:host]}/api/v1/accounts/1/sis_imports.json?import_type=instructure_csv")
```

Comment:
should change this to `https://#{opts[:host]}/api/v1/accounts/1/sis_imports.json?import_type=instructure_csv` to `https://#{opts[:host]}/api/v1/accounts/self/sis_imports.json?import_type=instructure_csv` in case the root account id is not 1